api/app.py

```diff
@@ -1,14 +1,24 @@
 from flask import Flask, redirect, url_for
+from flask_sqlalchemy import SQLAlchemy
 
 from api.admin import admin_bp
 from api.user import user_bp
 
+# db must be initialized before importing models
+db = SQLAlchemy()
+
+from api.user.comments import model
+from api.user.posts import model
+
 # note: Heroku will run things from the working directory as the root of this repo. Therefore, this path MUST
 # be relative to the root of the repo, NOT to this file. You will likely need to specify the working directory
 # as the root of this repo # when you run this file in your IDE
 app = Flask(__name__, static_folder="../frontend/build", static_url_path="/")
 app.register_blueprint(user_bp, url_prefix="/authors")
 app.register_blueprint(admin_bp, url_prefix="/admin")
+app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:////tmp/test.db"
```

Comment:
can this point to a location that local, eg ./test.db and then can we exclude that from gitignore. Just easier incase we need to eg manually inspect the db, then we know where it is and where to find it
```suggestion
app.config["SQLALCHEMY_DATABASE_URI"] = "sqlite:///local.db"
```